docs/architecture.md

```diff
@@ -0,0 +1,60 @@
+# Architecture
```

Comment:
There's also https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/#publishing-your-github-pages-site-from-a-docs-folder-on-your-master-branch, which we could leverage someday.